packages/framework-integration-tests/src/common/address.ts

```diff
@@ -1,8 +1,10 @@
-export interface Address {
-  firstName: string
-  lastName: string
-  country: string
-  state: string
-  postalCode: string
-  address: string
+export class Address {
```

Comment:
Why are these classes now?

Reply:
Because sadly right now the `interfaces`, `types`, and `TypeScript` custom classes are still lost at runtime, so only the user classes are being reflected. You can see the issue with the limitations that would be great to support in the future here #661.